src/self_driving/model_server/model_server_manager.cpp

```diff
@@ -69,6 +69,7 @@ ModelServerManager::ModelServerManager(const std::string &model_bin,
       case Callback::NOOP:
         break;
       case Callback::CONNECTED:
+        MODEL_SERVER_LOG_INFO("[PID={}] ModelServer connected", ::getpid());
```

Comment:
I kind of want to argue that this should be DEBUG or TRACE, but no strong enough opinion.

Just want to avoid having too many log messages when starting up the server, otherwise people will stop reading.